File: src/php/orders/create-order.php
<?php
require_once '/Users/jhammond/git/easypost/easypost-tools/vendor/autoload.php';
use EasyPost\EasyPost;
use EasyPost\Error;
use EasyPost\Address;
use EasyPost\Order;
use Dotenv\Dotenv;

$dotenv = Dotenv::createImmutable('/Users/jhammond/git/easypost/easypost-tools');
$dotenv->load();
EasyPost::setApiKey(getenv("EASYPOST_PROD_API_KEY"));

$to_address = Address::create(
    array(
        "street1" => "417 MONTGOMERY ST",
        "street2" => "FL 5",
        "city"    => "San Francisco",
        "state"   => "CA",
        "zip"     => "94104",
        "country" => "US",
        "company" => "EasyPost",
        "phone"   => "415-123-4567"
    )
);

$from_address = Address::create(
    array(
        "street1" => "417 MONTGOMERY ST",
        "street2" => "FL 5",
        "city"    => "San Francisco",
        "state"   => "CA",
        "zip"     => "94104",
        "country" => "US",
        "company" => "EasyPost",
        "phone"   => "415-123-4567"
    )
);

try {
    $order = Order::create(array(
        "to_address" => $to_address,
        "from_address" => $from_address,
        "shipments" => array(
            array(
                "parcel" => array(
                    // "predefined_package" => "FedExBox",
                    "weight" => 10.2
                )
            ),
            array(
                "parcel" => array(
                    // "predefined_package" => "FedExBox",
                    "weight" => 17.5
                )
            ),
        ),
        "carrier_accounts" => getenv("FEDEX"),
        "service" => "FEDEX_GROUND",
        "carrier" => "FedEx",
    ));
    echo $order;
} catch (Error $exception) {
    echo $exception;
}

File: src/python/orders/create_order.py
import easypost
import os
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_PROD_API_KEY')

order = easypost.Order.create(
    to_address={
        'name': 'Dr. Steve Brule',
        'street1': '179 N Harbor Dr',
        'city': 'Redondo Beach',
        'state': 'CA',
        'zip': '90277',
        'country': 'US',
        'phone': '4153334444',
        'email': 'dr_steve_brule@gmail.com'
    },
    from_address={
        'name': 'EasyPost',
        'street1': '417 Montgomery Street',
        'street2': '5th Floor',
        'city': 'San Francisco',
        'state': 'CA',
        'zip': '94104',
        'country': 'US',
        'phone': '4153334444',
        'email': 'support@easypost.com'
    },
    shipments=[
        {
            "parcel": {
                "predefined_package": "FedExBox",
                "weight": 10.2
            }
        },
        {
            "parcel": {
                "predefined_package": "FedExBox",
                "weight": 17.5
            }
        }
    ],
    carrier_account=[os.getenv('FEDEX')]
)

print(order)
